tracing-opentelemetry/src/layer.rs

```diff
@@ -612,6 +511,17 @@ mod tests {
             self.invalid()
         }
     }
+    impl PreSampledTracer for TestTracer {
```

Comment:
nit: should probably have a newline
```suggestion
    }
    
    impl PreSampledTracer for TestTracer {
```